src/signals/incident/components/ReplyForm/Notice/Notice.tsx

```diff
@@ -16,18 +11,28 @@ const StyledHeading = styled(Heading)`
 
 interface NoticeProps {
   title: string
-  content: string
+  content: string | string[]
 }
 
-const Notice = ({ content, title }: NoticeProps) => (
-  <Row>
-    <Column span={12}>
-      <div>
-        <StyledHeading>{title}</StyledHeading>
-        <Paragraph>{content}</Paragraph>
-      </div>
-    </Column>
-  </Row>
-)
+const Notice = ({ content, title }: NoticeProps) => {
+  const contentArray = Array.isArray(content) ? content : [content]
+
+  return (
+    <Row>
+      <Column span={12}>
+        <div>
+          <StyledHeading>{title}</StyledHeading>
+          {contentArray.map((item) => {
+            return (
+              <ReactMarkdown key={item} skipHtml allowedElements={['a', 'p']}>
```

Comment:
You're creating a new instance of `ReactMarkdown` here for every item. Why not just use Markdown as content? It allows for multiple paragraphs and such.